haystack/pipelines/base.py

```diff
@@ -390,8 +391,13 @@ def eval(
                         - Good default for multiple languages: "sentence-transformers/paraphrase-multilingual-mpnet-base-v2"
                         - Large, powerful, but slow model for English only: "cross-encoder/stsb-roberta-large"
                         - Large model for German only: "deepset/gbert-large-sts"
+            :param add_isolated_node_eval: Whether to additionally evaluate individual nodes based on labels as input instead of the output of the previous node in the pipeline
```

Comment:
As this is the core entry point for users, I'd extend the documentation here a bit more and:
- Answer the "Why" (understand the upper bound, understand if it makes sense to optimize a previous node or the individual one ...)
- Explain the result (will return more rows in the dataframe + the report has extra metrics ...)
- Link to the guide in documentation

Otherwise I am afraid that many users would not become aware that this is potentially helpful in their use case.